src/responsibleai/rai_analyse/_score_card/common_components.py

```diff
@@ -32,7 +32,7 @@ def get_full_html(htmlbody):
 def get_css():
     return """<style>
         * {
-          font-family: Segoe UI;
+          font-family: Ubuntu;
```

Comment:
nit: can you please add in description why this change is needed? like if PM requested it or are we keeping in compliance with something? thanks!

Reply:
Designer requested change of font for scorecard rendering.